app/services/queries/daily_metrics.rb

```diff
@@ -7,24 +7,24 @@ def initialize(project_id)
     end
 
     def call
-      users_project.map do |uspr|
-        metrics = uspr.metrics.where(created_at: (actual_time - FROM)..actual_time)
-        { name: uspr.user.login, data: format_data(metrics) }
-      end
+      ChartkickDataBuilder.call(
+        entity: users_project,
+        query: query
+      )
     end
 
     private
 
-    def users_project
-      @users_project ||= Project.find(@project_id).users_projects
+    def query
+      {
+        value_timestamp: (actual_time - FROM)..actual_time,
+        interval: :daily,
+        name: :review_turnaround
```

Comment:
the file is called `app/services/queries/daily_metrics.rb` , the name is pretty generic, so I don't think `name: :review_turnaround` belongs here. Shouldn't it be passed as a param to the service?

Reply:
yes, @hdamico is adding it in his PR 👍